src/actions/sessionActions.js

```diff
@@ -32,3 +44,22 @@ export const logout = () =>
     }).catch((err) => {
       throw (err);
     });
+
+export const editUser = user =>
+  () =>
+    sessionApi.updateUser(user)
+      .then(({ user }) => {
+        history.push(routes.index);
+        sessionService.saveUser(user);
+      })
+      .catch((err) => {
+        throw new SubmissionError({
+          _error: err.errors
+        });
+      });
+
+export const updateTopics = () => (dispatch) => {
```

Comment:
This function doesn't `update` topics, but rather `gets` them from the back-end. Can you rename this function?